### PR TITLE
remove gradle

### PR DESCRIPTION
**Pull request by flyinprogrammer**
flatten directories

move content into content- folders

update documentation based on changes

### CODE REVIEW

**Comment by viglucci** [comment]
Would be nice to land this into master than then rebase #29 on it since most of the changes are unrelated to #29 and im not sure when #29 will land.

Would you be able to change the PR base to master and just remove my work in progress page additions? I can worry about rebasing #29 onto master after that.

**Review by viglucci — CHANGES_REQUESTED** [review]
Would be nice to land this into master than then rebase #29 on it since most of the changes are unrelated to #29 and im not sure when #29 will land.

Would you be able to change the PR base to master and just remove my work in progress page additions? I can worry about rebasing #29 onto master after that.

**Comment by viglucci** [comment]
On second thought I'll make the changes to remove my work in-progress page. Should be minimal change, then we can change the PR base to master.

**Comment by viglucci** [comment]
> On second thought I'll make the changes to remove my work in-progress page. Should be minimal change, then we can change the PR base to master.

Addressed here: https://github.com/rsocket/rsocket-website/pull/40.

If we land that then we can land this in master without worrying about #29 while its in-progress.

**Comment by flyinprogrammer** [comment]
This is now ready for review again. I _believe_ when it lands in `master` the Github Action will work correctly and publish the site.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,37 +30,34 @@ When you're done making changes and you'd like to propose them for review, use o
...
-```
+    npm install
+    npm run start
 
 ### Verifying via GitHub Pages Deployment (Optional)
 
-You can test your changes live via a [GitHub Pages](https://pages.github.com/) deployment, which can be useful for demoing changes in a pull request, etc.
+TODO: @flyinprogrammer needs to fix this because it's all garbage once we start using Netlify.
+Starting with the fact that PROJECT_BASE_URL will likely become DEPLOY_URL from Netlify, but
+what will that value be for contributors or people who want to fork ¯\_(ツ)_/¯
```
Comment (viglucci) [inline CONTRIBUTING.md:43]: I think this `TODO` is now complete?

**Review by viglucci — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (flyinprogrammer) [reply CONTRIBUTING.md:43]: not really i was waiting for things to roll on to master so i could then use the fork->PR process to fix it :joy:

**Pull request merged**